.openpublishing.publish.config.json

```diff
@@ -12,7 +12,7 @@
       "locale": "en-us",
       "monikers": [],
       "moniker_ranges": ["common-js",
-                         "excel-js-preview", "excel-js-1.1", "excel-js-1.2", "excel-js-1.3", "excel-js-1.4", "excel-js-1.5", "excel-js-1.6", "excel-js-1.7", "excel-js-1.8", "excel-js-1.9",
+                         "excel-js-preview", "excel-js-1.1", "excel-js-1.2", "excel-js-1.3", "excel-js-1.4", "excel-js-1.5", "excel-js-1.6", "excel-js-1.7", "excel-js-1.8", "excel-js-1.9", "excel-js-1.10", "excel-js-online",
```

Comment:
Looks like you have the corresponding 1.10 moniker-related changes staged on the ref site (in branch `AlexJ-NewMonikers`); assuming you'll time the merging of this PR with merging of those changes to the ref docs.